### ISSUE TITLE
no such option: --strict

### ISSUE THREAD

**Issue by dlutsch** [mattrobenolt/jinja2-cli#33]
Running the below gives me an error about passing the strict option. I've tried including it both before the template name and after the input file but get an error in either case. 

```
jinja2 service.j2 api_gateway.yml --format=yml --strict
Usage: jinja2 [options] <input template> <input data>

jinja2: error: no such option: --strict
```

**Comment by mattrobenolt** [comment]
It sounds to me like you just have an old version before that flag was introduced. The latest is `0.5.0`, you can check with `jinja2 --version`.

**Comment by mattrobenolt** [comment]
Oh, weird. I guess I haven't tagged a new release in a while, and `--strict` didn't exist in 0.5.0. Will do that soon.

**Comment by mattrobenolt** [comment]
I've published 0.6.0 to pypi: https://pypi.python.org/pypi/jinja2-cli/0.6.0

Thanks for the nudge. :)

**Comment by dlutsch** [comment]
awesome, thanks for getting on that so quickly!
